src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirModel.cs

```diff
@@ -330,7 +329,7 @@ private void ThrowIfNotStarted()
         {
             if (!_started)
             {
-                _logger.LogError($"The {nameof(SqlServerModelInitializer)} instance has not been initialized.");
+                _logger.LogError($"The {nameof(SqlServerFhirModel)} instance has not been initialized.");
```

Comment:
FYI: I think this is also being done in #1142

Reply:
Thanks for pointing this out. I will close #1142